Sources/ApiVideoPlayer/ApiVideoPlayerController.swift

```diff
@@ -15,6 +16,19 @@ public class ApiVideoPlayerController: NSObject {
     private let taskExecutor: TasksExecutorProtocol.Type
     private let multicastDelegate = ApiVideoPlayerControllerMulticastDelegate()
     private var playerItemFactory: ApiVideoPlayerItemFactory?
+    private var infoNowPlaying: ApiVideoPlayerInformationNowPlaying
+    private let rcc = MPRemoteCommandCenter.shared()
+    private var enableRC = false {
+        didSet {
+            if enableRC {
+                self.setupRemoteControls()
+            } else {
+                #if !os(macOS)
+                UIApplication.shared.endReceivingRemoteControlEvents()
```

Comment:
In https://developer.apple.com/documentation/uikit/uiapplication/1623057-endreceivingremotecontrolevents:
```
In iOS 7.1 and later, use the shared [MPRemoteCommandCenter](https://developer.apple.com/documentation/mediaplayer/mpremotecommandcenter) object to unregister for remote control events. You do not need to call this method when using the shared command center object.
```

AFAIU, it is either you use rcc or `endReceivingRemoteControlEvents`